tests/sentry/integrations/github/test_client.py

```diff
@@ -17,7 +18,6 @@
 
 class GitHubAppsClientTest(TestCase):
     @mock.patch("sentry.integrations.github.client.get_jwt", return_value=b"jwt_token_1")
-    @responses.activate
```

Comment:
This was getting on the way of setting up the response for `https://api.github.com/app/installations/1/access_tokens`.

We can now remove it from every test.

```
        responses.add(
            method=responses.POST,
            url="https://api.github.com/app/installations/1/access_tokens",
            body='{"token": "12345token", "expires_at": "2030-01-01T00:00:00Z"}',
            content_type="application/json",
        )
```